tests/tracing/test_provider.py

```diff
@@ -112,3 +115,54 @@ def foo():
     # Re-enable tracing
     mlflow.tracing.enable()
     assert _is_enabled()
+
+
+@pytest.mark.parametrize("enable_mlflow_tracing", [True, False])
+def test_enable_mlflow_tracing_switch_in_serving_fluent(monkeypatch, enable_mlflow_tracing):
+    monkeypatch.setenv("ENABLE_MLFLOW_TRACING", str(enable_mlflow_tracing).lower())
```

Comment:
can we also test the behavior when `ENABLE_MLFLOW_TRACING` isn't set using `monkeypatch.delenv`?